x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/home/index.tsx

```diff
@@ -89,7 +89,7 @@ function mapToCard(
     title: item.title,
     version: 'version' in item ? item.version || '' : '',
     release: 'release' in item ? item.release : undefined,
-    uiInternalPathUrl,
+    url: uiInternalPathUrl,
```

Comment:
Prefer not to rename in the core-type to avoid cluttering this PR. Can do separately.